lib/hooks/useDragNDrop.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+
+const useDragAndDrop = () => {
+  const [highlight, setHighlight] = useState(false);
+
+  const fileListToArray = list => {
+    const array = [];
+
+    for (let i = 0; i < list.length; i += 1) {
+      array.push(list.item(i));
+    }
+    return array;
+  };
+
+  const onDragOver = evt => {
+    evt.preventDefault();
+    setHighlight(true);
+  };
+
+  const onDragLeave = e => {
+    e.preventDefault();
+    setHighlight(false);
+  };
+
+  const onDrop = event => {
+    event.preventDefault();
```

Comment:
Are you sure you need to do event.preventDefault(); ?? I am unable to drop any file on my machine